core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -210,6 +214,30 @@ paths:
         500:
           $ref: '#/components/responses/RawlsInternalError'
   /api/billing/v2:
+    get:
+      tags:
```

Comment:
Was missing. We will want to update orchestration Swagger also (both with deprecation and this). Waiting until after the tech talk to better understand how orchestration swagger and dispatching works.

Reply:
Created https://broadworkbench.atlassian.net/browse/WOR-521 for orchestration Swagger updates.